.circleci/config.yml

```diff
@@ -283,7 +313,7 @@ workflows:
           requires:
             - build_x86_64
             - build_aarch64
-      - integration_test:
+      - custom_networking_test:
```

Comment:
This is added only for testing purpose. I will remove it before marking this PR "read for review"